setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# setup.sh -- environment bootstrapper for python virtualenv
+
+set -euo pipefail
+
+SUDO=sudo
+if ! command -v $SUDO; then
+	echo no sudo on this system, proceeding as current user
+	SUDO=""
+fi
+
+if command -v apt-get; then
+	if dpkg -l python3-venv; then
+		echo "python3-venv is installed, skipping setup"
+	else
+		if ! apt info python3-venv; then
+			echo package info not found, trying apt update
+			$SUDO apt-get -qq update
+		fi
+		$SUDO apt-get install -qqy python3-venv
```

Comment:
I'm not sure what would happen if viam-server was not running already running as root -- I don't think a user would get a prompt for their password

Reply:
In my installation I get permission denied before reaching `setup.sh` so.....

Reply:
OK - looks you need to keep SUDO then